include/mbedtls/platform.h

```diff
@@ -196,11 +197,11 @@ int mbedtls_platform_win32_snprintf( char *s, size_t n, const char *fmt, ... );
 extern int (*mbedtls_snprintf)( char * s, size_t n, const char * format, ... );
 
 /**
- * \brief   Set your own snprintf function pointer
+ * \brief   This function sets your own snprintf function pointer.
  *
- * \param snprintf_func   the snprintf function implementation
+ * \param snprintf_func   The snprintf function implementation.
  *
- * \return              0
+ * \return              0.
```

Comment:
The above functions now use `\c 0 on success`.

Reply:
fixed